source/php.txt

```diff
@@ -137,7 +137,7 @@ MongoDB Compatibility
 
 .. include:: /includes/extracts/php-driver-compatibility-matrix-mongodb.rst
 
-.. include:: /includes/php-mongodb-compatibility-table.rst
+.. include:: /includes/mongodb-compatibility-table-php.rst
```

Comment:
This was the only reference I found to the file so I assume that there is no problem with renaming it to be consistent with the other driver includes.